client-app/shared/catalog/components/products-filters.vue

```diff
@@ -103,3 +161,11 @@ function onOpenBranches(): void {
   emit("openBranches");
 }
 </script>
+
+<style scoped lang="scss">
+:deep(.facet-filter--dropdown:last-child) {
+  .vc-popover__content {
+    @apply right-0 left-auto #{!important};
```

Comment:
For aligning the last (rightmost) filter dropdown to right and avoiding it from container overflowing.